srcpkgs/ardor/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'ardor'
+pkgname=ardor
+version=2.0.14
+revision=1
+wrksrc="${pkgname}"
+hostmakedepends="unzip"
+depends="openjdk"
```

Comment:
Is it a hard dependency on openjdk? Could it be replaced with `virtual?java-environment` (for oraclejdk) or even `virtual?java-runtime` if it doesn't need the development stuff?

Reply:
It more than likely should be `virtual?java-runtime`

Reply:
updated this and fixed some other bug (also for nxt)